client.go

```
@@ -97,10 +111,14 @@ type Statter interface {
 func Close() {
 	clientMux.RLock()
 	defer clientMux.RUnlock()
-	if client == nil {
-		return
+
+	if client != nil {
+		client.Close()
```

golangci-lint (2.12.1)

[error] Error return value of `client.Close` is not checked

(errcheck)